panel/src/components/Views/Pages/PageChangesView.vue

```diff
@@ -0,0 +1,180 @@
+<template>
+	<k-panel class="k-panel-inside k-page-changes-view">
+		<header class="k-page-changes-header">
+			<k-button-group>
+				<k-button
+					:link="link"
+					:responsive="true"
+					:text="$t('back')"
+					icon="angle-left"
+					size="sm"
+					variant="filled"
+				/>
+				<k-button
+					:icon="modes[mode].icon"
+					:dropdown="true"
+					:responsive="true"
+					:text="$t('view')"
```

Comment:
`Suggestion:` I find "View" very undescriptive and would prefer to show the currently selected mode here.

Reply:
I've tried that and found it confusing with the headlines below basically repeating the same. I also tried hiding the headlines and only showing them in the side-by-side view, but that also felt weird.

Reply:
Maybe, just a thought, taking inspiration from the previous experiment would work also here. If we move this button to the center to make space for the page title, we could try if instead of button with dropdown, we just have the three icons as a toggle, similar to the view size toggle we had in the other experiment:

<img width="166" alt="Screenshot 2024-10-22 at 4 13 21 PM" src="https://github.com/user-attachments/assets/9288502c-ba90-4ab7-9d38-9cee501c0212">

I could imagine that it doesn't actually need the text/headlines/labels at all and just the three icons next to each other make it quite clear what they actually do. Especially since then each section is labeled and self-explaining, as you say.